src/emulator/config/include/config/config_func.h

```diff
@@ -30,7 +30,7 @@ namespace app {
   * \param cfg Config operations to save.
   * \param output_path The location to save the configuration file.
   * \return Success on saving the config file, otherwise Error.
- */
+  */
```

Comment:
this change is requiert ?

Reply:
Simple alignment. It was off. I swear, Doxygen was happier with this.